docs/extensions/get-started/generator.md

```diff
@@ -0,0 +1,65 @@
+# New Extension Project with Generator
+
+The [Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) scaffolds a project ready for development. Install Yeoman and Lens Extension Generator with:
+
+```bash
+npm install -g yo generator-lens-ext
+```
+
+Run the generator and fill out a few fields for a TypeScript project:
+
+```bash
+yo lens-ext
+# ? What type of extension do you want to create? New Extension (TypeScript)
+# ? What's the name of your extension? my-first-lens-ext
+# ? What's the description of your extension? My hello world extension
+# ? What's your extension's publisher name? @my-org/my-first-lens-ext
```

Comment:
Is it possible for a `yo` generator to validate this field? Make sure that the user is following the `@<pub_name>/<name>` convention?

Reply:
Good point, it's possible, I will open an issue in generator repo https://github.com/lensapp/generator-lens-ext/issues/7